src/formpack/schema/datadef.py

```diff
@@ -23,15 +23,26 @@ class FormDataDef(object):
 
     def __init__(self, name, labels=None, has_stats=False, *args, **kwargs):
         self.name = name
+        self.unique_name = name
+        self.use_unique_name = False
         self.labels = labels or {}
-        self.value_names = self.get_value_names()
         self.has_stats = has_stats
 
     def __repr__(self):
-        return "<%s name='%s'>" % (self.__class__.__name__, self.name)
+        return "<%s name='%s'>" % (self.__class__.__name__, self.contextual_name)
```

Comment:
I have hesitated between using `self.name` for `FormDataDef` or changing the representation string because `contextual_name` is more related to `FormField` than its parent class.
Still wondering...